### PR TITLE
Display total charge amount on pos payment dialog

### PR DESCRIPTION
**Pull request by erdemyerebasmaz**
This PR addresses [POS-67](https://breeztech.atlassian.net/browse/POS-67)

"Scan the QR code to process this payment." text is replaced with "Scan to Process Payment" and moved to dialog's title.

Total charge amount is displayed in last selected currency(BTC/SAT) above QR image. If a fiat currency is selected, it's displayed right next to it in parenthesis.

### CODE REVIEW

**Comment by roeierez** [comment]
@erdemyerebasmaz can you merge from master to resolve conflicts?

**Comment by erdemyerebasmaz** [comment]
> @erdemyerebasmaz can you merge from master to resolve conflicts?

Resolved the conflicts https://github.com/breez/breezmobile/pull/311/commits/5195d2a2b49c34549a9fcfd6552836595f49f4ac

File: lib/routes/charge/pos_payment_dialog.dart [hunk 1]
```diff
@@ -61,41 +80,56 @@ class _PosPaymentDialogState extends State<PosPaymentDialog> {
...
-        ),
-        onPressed: () {
-          Navigator.of(context).pop(PosPaymentResult(clearSale: true));
-        },
-      );
+  @override
+  Widget build(BuildContext context) {
+    return AlertDialog(
+        titlePadding: EdgeInsets.fromLTRB(20.0, 22.0, 0.0, 8.0),
+        title: _syncedToChain
```
Comment (roeierez) [inline lib/routes/charge/pos_payment_dialog.dart]: Why use a class member for this that is assigned bellow in another component?
It is a lot cleaner if the account stream builder should be here on top and pass this value to the `buildWaitingPayment` function bellow.

Comment (erdemyerebasmaz) [reply lib/routes/charge/pos_payment_dialog.dart]: [Done](https://github.com/breez/breezmobile/pull/311/commits/82828126ea7fdab59a30b9c12e6138c79e4f9893)

**Pull request merged**